README.md

```diff
@@ -47,6 +47,10 @@ It is not recommended to bump fees when swapping because it can have unpredictab
 We are encourage community contributions whether it be a bug fix or an improvement to the documentation.
 Please have a look at the [contribution guidelines](./CONTRIBUTING.md).
 
+## Rust Version Support
+
+Please note that only the latest stable Rust toolchain is supported. All stable toolchains since 1.58 _should_ work.
```

Comment:
It is not a hard rule but we tend to put each sentence on a new line in markdown docs.
Works better together with git because git cares about lines and for markdown, a single line feed is ignored.


```suggestion
Please note that only the latest stable Rust toolchain is supported.
All stable toolchains since 1.58 _should_ work.
```